package.json

```diff
@@ -5,19 +5,24 @@
   "contributors": [
     "Irina, Catlin, Arup"
   ],
+  "engines": {
+    "node": ">=10.0.0 <13.0.0",
+    "npm": "6.9.0"
+  },
   "homepage": "",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build --build-optimizer --delete-output-path --optimization --progress --vendor-chunk --aot",
+    "build:prod": "npm run ng build -- --prod --configuration=deploy",
     "test": "jest --coverage",
     "test:dev": "jest --watch",
     "lint": "ng lint",
+    "lint:fix": "ng lint --fix",
     "e2e": "cypress open",
     "sonar-scanner": "npm run test && sonar-scanner",
     "test:deploy": "npm run build && npx http-server ./dist -p 4200 -c-1",
-    "prepush": "npm run build && npm test",
-    "tslint": "tslint src/**/*.ts",
+    "prepush": "npm test",
```

Comment:
I will suggest we still build locally during prepush. Sometimes the templates are broken during development and you do get the quick feedback of the broken template(s) during build. Though building locally is very slow. We can investigate why separately